src/main/java/com/efficio/fieldbook/web/common/controller/SampleTreeController.java

```diff
@@ -26,9 +15,7 @@
 
 import javax.annotation.Resource;
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
-import java.util.Map;
 
 /**
  * The Class SampleTreeController. <br/>
```

Comment:
We can update the TODO here (leaving the note here, it won't let me on the line below)
```java
TODO Migrate tree state logic to BMSAPI
```